src/test/java/uk/ac/ebi/embl/converter/cli/MainTest.java

```diff
@@ -16,9 +16,33 @@
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
 import org.junit.jupiter.api.Test;
+import picocli.CommandLine;
+import uk.ac.ebi.embl.converter.validation.RuleSeverity;
+import uk.ac.ebi.embl.converter.validation.ValidationRule;
 
 public class MainTest {
+
+    @Test
+    void testParseRules() {
+        String[] args = new String[] {"--rules=flatfile_no_ontology_feature:off"};
+        Map<ValidationRule, RuleSeverity> expected = new HashMap<>() {
+            {
+                put(ValidationRule.FLATFILE_NO_ONTOLOGY_FEATURE, RuleSeverity.OFF);
+            }
```

Comment:
Also add tests to throw / warn / ignore the following errors using RuleSeverityState
```
InvalidGFF3HeaderError
InvalidGFF3RecordError
UnmappedFFFeature
```